introduction.md

```diff
@@ -1,3 +1,22 @@
 Hello world, my name is CJ, and i wanna be a code ninja XD
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?